pkg/cluster/space/openebs_disk_space_validator.go

```diff
@@ -54,13 +54,19 @@ func (o *OpenEBSDiskSpaceValidator) NodesWithoutSpace(ctx context.Context) ([]st
 			continue
 		}
 
+		var reservedMsg string
+		if vol.RootVolume {
+			reservedMsg = "(15% of the root disk is reserved for the OS)"
```

Comment:
I would say instead that 15% is reserved to prevent diskpressure evictions

Reply:
Good. Done.